tslint.json

```diff
@@ -128,9 +128,6 @@
         "use-life-cycle-interface": true,
         "use-pipe-transform-interface": true,
         "component-class-suffix": true,
-        "directive-class-suffix": true,
-        "no-access-missing-member": true,
```

Comment:
The newest version of TSLint/Codelyzer removed these three rules.  My guess is they are obsolete, as the TS compiler has improved its checks.